apps/probate/probate-cron-hmrc-extract/demo/00.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   values:
     job:
       image: hmctspublic.azurecr.io/probate/back-office:pr-2422-de296cc-20231027134856 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
-      schedule: "0,20,40 * * * *"
+      schedule: "0 0 1-31/2 * *"
```

Comment:
@abbasiTahir Can you just copy and paste the schedule as on the story? This one with 0 0 will run at midnight and then the other would run at 2am. It just needs to run at same time.